### PR TITLE
Marketing 102 - Revamp of Why Netdata

### PR DESCRIPTION
**Pull request by shortpatti**
Simplified the "Why Netdata?" topic and I brought the tone to something that answers WHY without a lot of noise. Plus, I eliminated the unnecessary duplication within each topic so that it just answers the question.  I put this in ONE topic because I've seen the trend in tech docs to move away from individual topics and to a topic that covers ONE subject with sub-sections. This trend is meant to bring the content into a more cohesive flow and to reduce the number of clicking (drill downs) to get to an answer, keeping the user focused in one topic rather than jumping around to link after link. 

Updated the heading titles to the SECURITY.md topic. 

Removed the What is Netdata topic.

### CODE REVIEW

File: docs/why-netdata.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+| Requires **zero dedicated resources**|Require large dedicated resources|
+ 
+## Netdata's features
+Netdata is free, open-source software that currently runs on Linux, FreeBSD, and macOS.  It runs autonomously, without any third-party components, or it can integrate with existing monitoring toolchains, like Prometheus, Graphite, OpenTSDB, Kafka, Grafana, and more.
+ 
+We built Netdata around four principles:
+1.  [**1s granularity**](#1s-granularity). It is impossible to monitor a 2 second SLA, with 10-second metrics.
+2.  [**Unlimited metrics**](#unlimited-metrics). To troubleshoot slowdowns, we need all the available metrics.
+3.  [**Meaningful presentation**](#meaningful-presentation). The monitoring tool should know all the metrics. Users should not!
+4.  [**Immediate results**](#immediate-results). Just install and use it.
```
Comment (joelhans) [inline docs/why-netdata.md]: I'm not sure why, but this is being rendered as a single paragraph. Maybe you need a new line above the first item in the list?

```suggestion
We built Netdata around four principles:

1.  [**1s granularity**](#1s-granularity). It is impossible to monitor a 2 second SLA, with 10-second metrics.
2.  [**Unlimited metrics**](#unlimited-metrics). To troubleshoot slowdowns, we need all the available metrics.
3.  [**Meaningful presentation**](#meaningful-presentation). The monitoring tool should know all the metrics. Users should not!
4.  [**Immediate results**](#immediate-results). Just install and use it.
```

File: docs/generator/buildyaml.sh [hunk 2]
```diff
@@ -143,23 +143,10 @@ nav:"
...
-    - REDISTRIBUTED.md
-    - CHANGELOG.md
-    - SECURITY.md
-- Why Netdata:
-    - 'docs/why-netdata/README.md'
-    - 'docs/why-netdata/1s-granularity.md'
-    - 'docs/why-netdata/unlimited-metrics.md'
-    - 'docs/why-netdata/meaningful-presentation.md'
-    - 'docs/why-netdata/immediate-results.md'
+echo -ne " - 'docs/why-netdata.md'
```
Comment (joelhans) [inline docs/generator/buildyaml.sh]: There's a Netlify error due to this one line, just one missing space! Finicky YAML...

```suggestion
echo -ne "  - 'docs/why-netdata.md'
```

File: docs/why-netdata.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+*    Netdata interpolates collected metrics to eliminate the error introduced by data collection latencies on busy virtual environments. It does this using microsecond timings, per data source, offering measurements with an error rate of 0.0001%. When running [in debug mode, Netdata calculates this error rate](https://github.com/netdata/netdata/blob/36199f449852f8077ea915a3a14a33fa2aff6d85/database/rrdset.c#L1070-L1099) for every point collected, ensuring that the database works with acceptable accuracy.
+
+*    Netdata is just FAST because optimization is a core product feature. On modern hardware, Netdata can collect metrics with a rate of above 1M metrics per second per core (this includes everything, parsing data sources, interpolating data, storing data in the time-series database, etc.). So, for a few thousand metrics per second per node, Netdata needs negligible CPU resources (just 1-2% of a single core). 
+
+So, for Netdata 1s granularity is easy, the natural outcome!
```
[the remark this answers is not in the document]
Comment (shortpatti) [reply docs/why-netdata.md]: Probably not.

File: docs/why-netdata.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
```
Comment (shortpatti) [inline docs/why-netdata.md]:
```suggestion

```

**Comment by shortpatti** [comment]
Demo sites -> See a Netdata demo
Security design -> **???**
Anonymous statistics -> Running Netdata
Donations -> Additional information
Redistributed software -> Additional information
Security Policy -> Additional information

File: docs/generator/buildyaml.sh (hunk 2 above)
Comment (joelhans) [reply docs/generator/buildyaml.sh]: Sadly it looks like your merge removed this space, causing Netlify to fail again. 😢

**Comment by shortpatti** [comment]
Yeah, I’m working on this headache 😊  

 

Thank you,

 

Patti Short

Technical Writer

 



Monitor everything... in real-time!

 <https://www.netdata.cloud/> https://www.netdata.cloud

**Comment by joelhans** [comment]
> Demo sites -> See a Netdata demo
> Security design -> **???** @joelhans I think you said you were doing something with this and you told me to put it at the root of /docs
> Anonymous statistics -> Running Netdata
> Donations -> Additional information
> Redistributed software -> Additional information
> Security Policy -> Additional information

Sorry, I missed the demo sites somehow. My bad. And yes, I'm working on the security stuff early next week.

I think this is ready to go once the Netlify error is fixed!

**Pull request closed without merging**